docs/index.md

```diff
@@ -1,10 +1,56 @@
 ---
 title: nf-cmgg/structural
-description: A bioinformatics best-practice analysis pipeline for calling structural variants from short reads.
+description: A bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads
```

Comment:
structural variants (SVs), copy number variants (CNVs) and repeat region expansions (RREs)